### DISCUSSION TITLE
Basic Python

### DISCUSSION THREAD

**Discussion by myibra** [teamalgoritma/community#25] [Q&A] [answered]
Bagaimana melakukan penjumlahan array menggunakan for loop?
Contoh:
ar = [1, 2, 3]
1+2+3 = 6
Hasil = 6

**Comment by triarm** [comment, marked as answer]
Untuk melakukan penjumlahan pada array menggunakan for dapat dilakukan seperti berikut
```
hasil = 0
for element in ar:
    hasil = hasil + element
```

Selain itu, untuk penjumlahan array, Bapak dapat langsung menjumlahkannya dengan fungsi `sum()`, yaitu:
```
sum(ar)
```
